src/DynamoCoreWpf/ViewModels/PackageManager/PackagePathViewModel.cs

```diff
@@ -226,10 +226,15 @@ private void RemovePathAt(int index)
 
         private void CommitChanges(object param)
```

Comment:
Is this method called every time the preferences dialog is closed even though the package preferences tab isn't touched?

Reply:
yes
https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs#L94